src/Umbraco.Web/Routing/DomainUtilities.cs

```diff
@@ -264,7 +262,7 @@ private static DomainAndUri GetByCulture(IReadOnlyCollection<DomainAndUri> domai
                 if (domainAndUri != null) return domainAndUri;
             }
 
-            return domainsAndUris.First(); // what else?
+            return domainsAndUris.FirstOrDefault(); // what else?
```

Comment:
Although 0 domains is already handled within `SelectDomain()` (returning null), let's just ensure this does the same...